rancher/charts
# fix(Longhorn): fix group name for Longhorn Longhorn CSI Liveness Prob…

## Issue: 
https://github.com/longhorn/longhorn/issues/5794

## Problem
The group name of `Longhorn CSI Liveness Probe Image Tag` in `question.yaml` is not correct and it makes the options category to the wrong page.

## Solution
Fix the upstream to make the group name should be Longhorn CSI Driver Images rather than Longhorn CSI Liveness Probe Images

## Testing
 1. Having a K8S cluster
 2. Add repository with https://github.com/weizhe0422/rancher-charts/tree/dev-v2.7_LH_Issue%235794
 3. Try to install Longhorn v1.3.3 and check the `Edit Options
 4. Uncheck `Use Default Images`
 5. The 'Longhorn CSI Liveness Probe Image Tag' should belong to `Longhorn CSI Driver Images` page like below 
    <img width="1067" alt="截圖 2023-04-25 下午2 56 33" src="https://user-images.githubusercontent.com/17548901/234198148-2f0f09fc-e867-4eef-809f-0e6eca7acc8b.png">

cc @innobead @PhanLe1010 @khushboo-rancher 

Signed-off-by: Ray Chang <ray.chang@suse.com>
